Unix/installbuilder/GNUmakefile

```diff
@@ -200,16 +214,19 @@ ifneq ($(PF),Darwin)
 		--RELEASE=$(CONFIG_PATCH_LEVEL) \
 		$(OUTPUTFILE_LINE_WITHSYMBOLS) \
 		$(SSL_BUILD_LINE) \
-		--DPKG_LOCATION=$(TOP)/../../pal/installer/InstallBuilder/tools/bin/dpkg-deb-$(PF_ARCH) \
+		$(DPKG_LOCATION) \
 		--DATAFILE_PATH=$(TOP)/installbuilder/datafiles \
 		$(DATAFILES) Linux_DPKG.data
     endif # ifeq ($(BUILD_DPKG),1)
 
+    ifneq ($(ENABLE_DEBUG),1)
```

Comment:
This one conditional can be combined with conditionals below. Or, said another way, if we're not DEBUG, then do lines 228-275. Or something like that.

Reply:
Done